### ISSUE TITLE
PTZ

### ISSUE THREAD

**Issue by marcoherbst** [evercam/ex_nvr#74]
Implement OnVIF PTZ functionality.

I'm flagging this as a good first issue because I suspect a lot of the principles already exist in Evercam Backend (and that was done by an Elixir Newbie ... see video posted online).

Add UI PTZ dashboard
<img width="950" alt="image" src="https://github.com/evercam/ex_nvr/assets/51049639/b51d340e-eb95-4ae9-9e6f-f968d5049ef6">

to ExNVR dashboard
<img width="934" alt="image" src="https://github.com/evercam/ex_nvr/assets/51049639/a18b8e1c-d37c-4249-8552-e3d6ab7cbaa8">

NB: only display the buttons when camera supports PTZ

**Comment by marcoherbst** [comment]
I'd suggest doing only the very least possible effort on UI as @KamelBadji as some ideas up his sleeve.

**Comment by marcoherbst** [comment]
Thought the mermaid overlay is quite a nice UI 

![Image](https://github.com/user-attachments/assets/c639e610-a5e5-4839-a697-a541a187b369)

**Comment by gBillal** [comment]
@salah-walid what about you work on this ?

**Comment by salah-walid** [comment]
@gBillal yes, but i would wait until [this PR](https://github.com/evercam/ex_nvr/pull/606) is merged to add on it

**Comment by gBillal** [comment]
@salah-walid just have the UI component ready. We still need back-end code for it to function correctly. This is a low priority task.

**Comment by azharmalik3** [comment]
@gBillal I'm willing to work on the PTZ backend code.

**Comment by gBillal** [comment]
@azharmalik3 I'll assign you the issue when the time comes, there's some stuff to fix before.
